include/mbedtls/x509_crl.h

```diff
@@ -96,6 +97,18 @@ typedef struct mbedtls_x509_crl
 mbedtls_x509_crl;
 
 /**
+ * \brief          Compare two CRL entry objects
+ *
+ * \param arg1     Pointer to the first entry to compare
+ * \param arg2     Pointer to the second entry to compare
+ *
+ * \return         an integer less than, equal to, or greater than zero if the
+ *                 first entry is respectively smaller than, equal to or greater
+ *                 than the second entry
+ */
+int mbedtls_x509_crl_entry_cmp( const void *arg1, const void *arg2);
```

Comment:
I would have liked to use some internal header and avoid exposing this function to the public API, but I am not sure there is any way.
Unless we move the whole mbedtls_x509_crt_is_revoked() function to library/x509_crl.c and expose it. Would this be viable?